apollo-api/src/main/java/com/apollographql/apollo/api/Error.kt

```diff
@@ -0,0 +1,74 @@
+package com.apollographql.apollo.api
+
+/**
+ * Represents an error response returned from the GraphQL server
+ */
+data class Error(
+    /**
+     * Server error message
+     */
+    @JvmField
+    val message: String,
```

Comment:
Is the `@JvmField` required here ? It exposes a field publicly for no apparent reason.